### PR TITLE
Cirrus: Adjust destination branch name

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM once things go green

File: .cirrus.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ gating_task:
     pipefail_enabledscript: 'if /bin/false | /bin/true; then echo "pipefail fault" && exit 72; fi'
 
     on_failure:
-        failed_master_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_master_failure.sh'
+        failed_v1.4.2-stable_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_v1.4.2-stable_failure.sh'
```
Comment (cevich) [inline .cirrus.yml]: Woops.

**Pull request merged**